depgenerators/maven.req

```diff
@@ -308,10 +312,7 @@ class TagBuilder(object):
         return req
 
     def _process_java_version(self, version_string):
-        version = version_string.split('.')
-        major = int(version[0])
-        minor = int(version[1])
-        return "1:{0}.{1}".format(major, minor)
+        return "1:{0}".format(version_string)
```

Comment:
Could this be inlined? IMO, it would be more readable than the generic function name